packages/bpk-component-modal/src/BpkModalDialog-test.js

```diff
@@ -19,19 +19,30 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 
-/* eslint-disable import/first */
 import BpkModalDialog from './BpkModalDialog';
-/* eslint-enable */
 
 describe('BpkModalDialog', () => {
+  let closeEvents;
+  beforeAll(() => {
+    closeEvents = {
+      onTouchStart: jest.fn(),
+      onTouchMove: jest.fn(),
+      onTouchEnd: jest.fn(),
+      onMouseDown: jest.fn(),
+      onMouseMove: jest.fn(),
+      onMouseUp: jest.fn(),
+    };
+  });
   it('should render correctly', () => {
     const tree = renderer.create(
       <BpkModalDialog
         id="my-modal"
         title="Modal title"
         onClose={() => null}
```

Comment:
`jest.fn()` here will be more in line with the rest of the test file

Reply:
done